### DISCUSSION TITLE
github copilot only suggest one line

### DISCUSSION THREAD

**Discussion by AlfinIndrawan** [community/community#31041] [Copilot Conversations]
<img width="331" alt="Screenshot 2022-08-27 at 21 50 59" src="https://user-images.githubusercontent.com/77723289/187035462-eb6e01ef-9868-473f-9a07-68a3907feba1.png">
guys when i tried my github copilot for the first time, it suggest me multi line. but now every time i use it, it only suggest one line including when i ask a function. already tried to reinstall but dont work. im using macbook 2019. i have seen old post that have this problem but they said they already fixed it. anyone know solutions

**Comment by CiberNin** [comment]
I think this change was intentional and I don't really disagree. I find that one line completions are much more likely to be something I actually  want, and multi-line completions cause alot more reflow on the screen which can be disorienting to some users.
